examples/support/mock_data.py

```diff
@@ -65,20 +65,26 @@ def generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, Delta_SO, zs
     cosmo : dict
         Dictionary of cosmological parameters. Must contain at least, Omega_c, Omega_b,
         and H0
-    Delta_SO : float
-        Overdensity density contrast used to compute the cluster mass and concentration. The
-        spherical overdensity mass is computed as the mass enclosed within the radius
-        :math:`R_{\Delta{\rm SO}}` where the mean density is :math:`\Delta_{\rm SO}` times
-        the mean density of the Universe at the cluster redshift
-        :math:`M_{\Delta{\rm SO}}=4/3\pi\Delta_{\rm SO}\rho_{m}(z_{\rm lens})R_{\Delta{\rm SO}}^3`
     zsrc : float or str
         Choose the source galaxy distribution to be fixed or drawn from a predefined distribution.
         float : All sources galaxies at this fixed redshift
         str : Draws individual source gal redshifts from predefined distribution. Options
               are: chang13
+    Delta_SO : float, optional
+        Overdensity density contrast used to compute the cluster mass and concentration. The
+        spherical overdensity mass is computed as the mass enclosed within the radius
+        :math:`R_{\Delta{\rm SO}}` where the mean density is :math:`\Delta_{\rm SO}` times
+        the mean density of the Universe at the cluster redshift
```

Comment:
Maybe it's better to add parentheses: the mean (matter) density?

Reply:
Thanks, done.